src/main/resources/default/assets/scripts/client-pagination.js

```diff
@@ -9,22 +9,27 @@
 function Pagination(_paginationContainer, pageSize, update) {
     const self = this;
 
+    _paginationContainer.classList.add('d-flex', 'justify-content-center');
     _paginationContainer.innerHTML = '';
 
     const _paginationGroup = document.createElement('div');
     _paginationGroup.classList.add('input-group')
-    _paginationGroup.classList.add('justify-content-center')
     _paginationGroup.classList.add('mb-4')
+    _paginationGroup.style.width = 'fit-content';
     _paginationContainer.appendChild(_paginationGroup);
 
-    this._previousBtn = document.createElement("div");
-    this._previousBtn.classList.add('input-group-prepend');
-    this._previousBtn.innerHTML = "<a href='#' class='btn btn-outline-secondary' aria-label='Previous'><span aria-hidden='true'>&#8592;</span></a>";
-    this._nextBtn = document.createElement('div');
-    this._nextBtn.classList.add('input-group-append');
-    this._nextBtn.innerHTML = "<a href='#' class='btn btn-outline-secondary' aria-label='Next'><span aria-hidden='true'>&#8594;</span></a>";
+    this._previousBtn = document.createElement("a");
+    this._previousBtn.href = '#';
+    this._previousBtn.classList.add('btn', 'btn-outline-secondary');
+    this._previousBtn.ariaLabel = 'Previous';
+    this._previousBtn.innerHTML = "<span aria-hidden='true'>&#8592;</span>";
+    this._nextBtn = document.createElement('a');
+    this._nextBtn.href = '#';
+    this._nextBtn.classList.add('btn', 'btn-outline-secondary');
+    this._nextBtn.ariaLabel = 'Next';
```

Comment:
isn´t there an NLS-key?

Reply:
Not yet, but introduced one along with ARIA support for the static pagination in https://github.com/scireum/sirius-web/pull/1379/commits/c58dec18a774e5d6f7f611c08d79809bd601530a